internal/rpmmd/repository.go

```diff
@@ -32,7 +32,7 @@ type Package struct {
 
 type PackageSpec struct {
 	Name    string `json:"name"`
-	Epoch   uint   `json:"epoch,omitempty"`
+	Epoch   uint   `json:"epoch"`
```

Comment:
The `omitempty` was for not sending anything when the `Epoch` was unset (i.e., 0). Why did you have to remove it? Maybe this points to a deeper problem somewhere where we forget to set it?

Reply:
lorax-composer seems to always send an epoch of 0 if there is no epoch defined. This seems to follow the behaviour outlined [here](https://weldr.io/RPM-Dependencies/) which states:

> If the epoch is not specified, it’s treated as 0.

If I run `dnf info 2ping` I see that 2ping does not have an epoch defined. When I call lorax-composer's /projects/info/2ping its build contains "epoch: 0". This seems to come from the python3-dnf plugin lorax uses which will return 0 for the epoch of a package with no defined epoch. Epoch is not displayed by cockpit-composer but is checked for in the props validation. For equivalence with lorax we should default to an epoch of 0 but not returning an epoch does not break anything in the UI.